docs/src/cookbooks/cookbook.md

```diff
@@ -445,7 +445,7 @@ import scala.collection.immutable.ListMap
 
 ```scala mdoc:crash
 class CustomBundleBroken(elts: (String, Data)*) extends Record {
-  val elements = ListMap(elts: _*)
+  override val elements = ListMap(elts: _*)
```

Comment:
these `override`s are not strictly required and could be removed in this PR.

Reply:
Alternatively, stylistically I encourage people to use `override` when implementing a virtual method even though it isn't required. We didn't use to encourage that but I think doing so may be a good idea. Potentially a separate change from this PR though since we should probably make all of the code consistent in that regard (there might even be a lint check or custom scalafix rule for it we can later enable).